terraform-consul-provider/overview.md

```diff
@@ -0,0 +1,128 @@
+## Overview of Workspace
+
+This is the workspace for the [**Register External Services with Terraform**](http://learn.hashicorp.com/consul/developer-discovery/terraform-consul-provider) guide.
+
+These services can be open by clicking on their respective tabs in the terminal.
+
+On initialization, this workspace should have ran `docker-compose up -d` which spins
+up the above services in the background. To view these instances, run `docker ps`{{execute}}
+
+It takes about a minute after you see the `Ready` message for the Consul datacenter
+to nominate a cluster leader and for you to continue to the next step.
+
+## Bootstrap Consul datacenter
+
+`docker exec -it consul-playground_consul-server-1_1 consul acl bootstrap`{{execute}}
+
+For more information about this command, please reference the guide.
+
+## Localhost Substitution
+
+NOTE: If you are using Katacoda, Katacoda uses a unique method to route its services.
+
+As a result, you should substitute any references to localhost to the following:
+
+| Learn Address   | Katacoda Address |
+| --------------  | ---------------- |
+| localhost:8500  | `[[HOST_SUBDOMAIN]]-8500-[[KATACODA_HOST]].environments.katacoda.com` |
+| localhost:9001  | `[[HOST_SUBDOMAIN]]-9001-[[KATACODA_HOST]].environments.katacoda.com` |
+| localhost:8080  | `[[HOST_SUBDOMAIN]]-8080-[[KATACODA_HOST]].environments.katacoda.com` |
+
+Remember to substitute `ACL_TOKEN_HERE` with your master ACL token. Your `main.tf` file should be similar to:
```

Comment:
Alternatively you can copy the following `main.tf` file, which represents the finished state of the guide and has the correct addressing. Remember to substitute `ACL_TOKEN_HERE` with your master ACL token. 

If you use this file you will only need to run `terraform apply`{{execute}} once to see all the outputs described in the guide.